reference/commands/misc/remote.rst

```diff
@@ -144,6 +146,13 @@ the command will insert the remote in the specified position
 
       $ conan remote update_ref OpenSSL/1.0.2i@conan/stable local-remote
 
+- Enable or disable remotes (accepts patterns such as * as argument):
```

Comment:
```suggestion
- Enable or disable remotes (accepts patterns such as ``*`` as argument):
```

Reply:
And also mention we are using `fnmatch` inside

Reply:
@czoido take a look at the comment above